### ISSUE TITLE
Mast cell tryptase level update

### ISSUE THREAD

**Issue by xirasasa** [obophenotype/human-phenotype-ontology#5597]
Replace:
“increased or decreased serum mast cell beta tryptase concentration”
With 2 new ones
 
“Increased/elevated basal serum tryptase”
 
“transient increase in total serum tryptase”
 
Please see explanation  for this request below from an experienced mast cell and tryptase expert.

"There is no way to measure beta tryptase…which is what this term reports.  And there is no such thing as “low” or “decreased” tryptase…the lower limit of normal is undetectable.  Finally, tryptase is produce by more than mast cells, thus ever reported “mast cell tryptase” is likewise false."

**Comment by pnrobinson** [comment]
This comment is referring to this term
https://hpo.jax.org/app/browse/term/HP:0031900

**Comment by pnrobinson** [comment]
Update from Sandhya,

Preferred term label:
Elevated total serum tryptase

Synonyms

Definition (free text, please give PubMed ID)
The presence of elevated tryptase may occur acutely due to mast cell degranulation.  Alternatively, chronic elevation of basal serum tryptase may be present in individuals with: 1) clonal myeloid neoplasms; 2) the genetic trait hereditary alpha-tryptasemia; 3) chronic kidney disease

Comment: Serum tryptase levels are useful in the diagnosis of mast cell activation, clonal myeloid disease, and hereditary alpha-tryptasemia.

PMID: 27749843  PMCID: PMC5397297  DOI: 10.1038/ng.3696

PMID: 3295549  DOI: 10.1056/NEJM198706253162603

PMID: 19522836  DOI: 10.1111/j.1365-2362.2009.02184.x

PMID: 16372800  DOI: 10.1111/j.1610-0387.2005.05706.x

Parent term (use hpo.jax.org/app)
No idea – Sandhya perhaps you can help me here?

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
hereditary alpha-tryptasemia, mastocytosis, monoclonal mast cell activation syndrome, AML, APML, JMML, CMML, CML, MDS, mast cell leukemia, myeloid hypereosinophilic syndrome, chronic eosinophilic leukemia

Also seen in CKD and ESRD

**Comment by pnrobinson** [comment]
corrected as advised. There is now one term, Elevated total serum tryptase
